alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/job/details/JiraServerJobDetailsModel.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.persistence.model.job.details;
+
+public class JiraServerJobDetailsModel extends DistributionJobDetailsModel {
+    private final boolean addComments;
+    private final String issueCreatorUsername;
+    private final String projectNameOrKey;
+    private final String issueType;
+    private final String resolveTransition;
+    private final String reopenTransition;
+
+    public JiraServerJobDetailsModel(boolean addComments, String issueCreatorUsername, String projectNameOrKey, String issueType, String resolveTransition, String reopenTransition) {
+        super("channel_jira_server");
```

Comment:
We should get this key from the Jira Server Channel Key. Either construct the Object here and call getUniversalKey() OR make this string public static final in the Jira Server Channel Key.

Reply:
Unfortunately, the specific channel keys are not available here. I added a TODO in the base model for this purpose.